client/capital/index.tsx

```diff
@@ -0,0 +1,211 @@
+import Chip from 'components/chip';
+import { useLoans } from 'wcpay/data';
+
+import './style.scss';
+
+const columns = [
+	{
+		key: 'paid_out_at',
+		label: __( 'Dispursed', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Dispursed', 'woocommerce-payments' ),
+		required: true,
+		isLeftAligned: true,
+		defaultSort: true,
+	},
+	{
+		key: 'status',
+		label: __( 'Status', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Status', 'woocommerce-payments' ),
+		required: true,
+		cellClassName: 'is-center-aligned',
+	},
+	{
+		key: 'amount',
+		label: __( 'Amount', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Amount', 'woocommerce-payments' ),
+		required: true,
+		isLeftAligned: false,
+	},
+	{
+		key: 'fee_amount',
+		label: __( 'Fixed fee', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Fixed fee', 'woocommerce-payments' ),
+		required: true,
+		isLeftAligned: false,
+	},
+	{
+		key: 'withhold_rate',
+		label: __( 'Withhold rate', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'Withhold rate', 'woocommerce-payments' ),
+		required: true,
+		isLeftAligned: false,
+	},
+	{
+		key: 'first_paydown_at',
+		label: __( 'First paydown', 'woocommerce-payments' ),
+		screenReaderLabel: __( 'First paydown', 'woocommerce-payments' ),
+		required: true,
+		isLeftAligned: false,
+	},
+];
+
+const getLoanStatusText = ( loan: CapitalLoan ) => {
+	return loan.fully_paid_at
+		? __( 'Paid off', 'woocommerce-payments' ) +
+				': ' +
+				dateI18n( 'M j, Y', loan.fully_paid_at )
+		: __( 'Active', 'woocommerce-payments' );
+};
+
+const getLoanStatusChip = ( loan: CapitalLoan ) => {
+	return (
+		<Chip
+			message={ getLoanStatusText( loan ) }
+			type={ loan.fully_paid_at ? 'primary' : 'warning' }
+		/>
+	);
+};
+
+const getRowsData = ( loans: CapitalLoan[] ) =>
+	loans.map( ( loan ) => {
+		const clickable = ( children: React.ReactNode ) => (
+			<ClickableCell
+				href={
+					'/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Ftransactions' +
+					'&type=charge&filter=advanced&loan_id_is=' +
+					loan.stripe_loan_id
+				}
+			>
+				{ children }
+			</ClickableCell>
+		);
+
+		const data = {
+			paid_out_at: {
+				value: loan.paid_out_at,
+				display: clickable( dateI18n( 'M j, Y', loan.paid_out_at ) ),
+			},
+			status: {
+				value: getLoanStatusText( loan ),
+				display: clickable( getLoanStatusChip( loan ) ),
+			},
+			amount: {
+				value: isZeroDecimalCurrency( loan.currency )
+					? loan.amount
+					: loan.amount / 100,
+				display: clickable(
+					formatExplicitCurrency( loan.amount, loan.currency )
+				),
+			},
+			fee_amount: {
+				value: isZeroDecimalCurrency( loan.currency )
+					? loan.fee_amount
+					: loan.fee_amount / 100,
+				display: clickable(
+					formatExplicitCurrency( loan.fee_amount, loan.currency )
+				),
+			},
+			withhold_rate: {
+				value: loan.withhold_rate,
+				display: clickable( loan.withhold_rate + '%' ),
+			},
+			first_paydown_at: {
+				value: loan.first_paydown_at,
```

Comment:
Argh, all my loans had paydowns already. On , I've made it so it renders a "-" instead of "Invalid date". Whatever we decide the long-term fix for this is, it will be a server-side-only change.

@LevinMedia Any thoughts on this? "First paydown [date]" has a different meaning in the summary (the date when Stripe will start taking a cut from transactions) and in the loans list (the date of the first paydown). Should we change the label to be clear that those 2 are different dates, or change one of them to fit the other?